test/unsafe_methods_test.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "minitest/autorun"
+require "rgeo/shapefile"
+
+class UnsafeMethodsTest < Minitest::Test
+  def test_unsafe_methods_true
+    _open_shapefile("test", true) do |file_|
```

Comment:
I don't know that we want to go on with those `_method` rather than `method` and `var_` rather than `var` @keithdoggett ? 

Although for the current PR it is ok with me, this is not the largest file to change when/if need be!

Reply:
Thought it was weird but it conforms to current practice. I can rename in a follow up PR if you'd like!

Reply:
Well sure if you want to contribute I'm all in. @keithdoggett is it ok with you?